network_simulator/packet.py

```diff
@@ -0,0 +1,76 @@
+"""
+This module will hold the class Packet, subclasses of Packet, and various Packet-related Events.
```

Comment:
It looks like you're exceeding the 80-char limit a lot in this file. You should see a line at the 80-char mark by default in PyCharm, and you can enforce wrapping (when typing reaches this margin) by going to Settings --> Editor --> Code Style in PyCharm.